A Step-by-Step Guide to Deploying and Operating an Acoustic Modem running UnetStack to Achieve the Best Performance

manuignatius opened the pull request:
Blog: https://github.com/org-arl/unet-blog/blob/demo/_posts/2023-04-03-A-Step-by-Step-Guide-to-Deploying-and-Operating-an-Acoustic-Modem-running-UnetStack-to-Achieve-the-Best-Performance.md

mchitre reviewed (changes requested):
Overall good, but a few things to address:

- [x] You're missing YAML headers.
- [x] There's a lot of mention of gen3 and gen4 modems. Is this a UnetStack blog or a Subnero modem blog? Decide and be consistent. If you use gen3/gen4 tags, decide on capitalization and naming and be consistent. There's also mention of gen3 modems with UnetStack 4 (speedtest loaded by default) -- which is obviously incorrect. Fix.
- [ ] You may want to have file transfer before (and after) tuning the link. Otherwise it gives the feel that there are many many steps involved before you can transfer a file.
- [ ] You may want to mention expectation of 50% link rate for speedtest before you show the numbers, so that the first impression is not "oh, so low?"

notthetup commented on _posts/2023-04-03-A-Step-by-Step-Guide-to-Deploying-and-Operating-an-Acoustic-Modem-running-UnetStack-to-Achieve-the-Best-Performance.md:
```diff
@@ -0,0 +1,173 @@
+
+> The `CONTROL` channel is meant for low-rate robust data transmission, whereas the `DATA` channel is typically configured for higher-rate data transmission. You can read more about these in the [unet handbook](https://unetstack.net/handbook).
+ 
+However, most of the actual payload transmissions are done using the `DATA` channel. So, our next step is to verify that the `DATA` channel works fine between the modems. This can be achieved using the `tell` command. Type the following command in the web shell of modem A.
```
Maybe we can also note that `tell` uses a combination of both `CONTROL` and `DATA` channel?

manuignatius commented:
> Overall good, but a few things to address:
> 
> * [x]  You're missing YAML headers.
> * [x]  There's a lot of mention of gen3 and gen4 modems. Is this a UnetStack blog or a Subnero modem blog? Decide and be consistent. If you use gen3/gen4 tags, decide on capitalization and naming and be consistent. There's also mention of gen3 modems with UnetStack 4 (speedtest loaded by default) -- which is obviously incorrect. Fix.
> * [x]  You may want to have file transfer before (and after) tuning the link. Otherwise it gives the feel that there are many many steps involved before you can transfer a file.
> * [x]  You may want to mention expectation of 50% link rate for speedtest before you show the numbers, so that the first impression is not "oh, so low?"

Addressed all comments. For the second comment, since we have backported speedtest to UnetStack 3.x and did a release, I assumed that it is going to be standard on the latest UnetStack v3 release onwards. Removed all references to gen3/4 modems as they are redundant.
